seed/views/organizations.py

```diff
@@ -460,14 +464,15 @@ def remove_user(self, request, pk=None):
         ou.delete()
 
         # check the user and make sure they still have a valid organization to belong to
-        if request.user.default_organization == org:
-            # find the first org and set it to that. It is okay if first_org is none.
-            # it simply means the user has no allowed organizations and will need an admin to
-            # assign them to a new organization if they want to use the account again.
-            first_org_user = OrganizationUser.objects.filter(user=user).order_by('id').first()
-            if first_org_user:
-                request.user.default_organization = first_org_user.organization
-                request.user.save()
+        user_orgs = OrganizationUser.objects.filter(user=user)
+
+        if user_orgs.count() == 0:
+            # Remove orphaned user
+            user.delete()
```

Comment:
are we sure we want to delete the user? Or should we just deactivate their account? If we deactivate the account, then we would need to make sure that the is_active flag is respected throughout the code. My main concern is that there are models that reference user ids (e.g., notes)

https://github.com/SEED-platform/seed/blob/4587ae1e8a1eec589677086ce1c887898521d13b/seed/landing/models.py#L53-L56